packages/connect/src/api/ethereum/ethereumDefinitions.ts

```diff
@@ -0,0 +1,53 @@
+import fetch from 'cross-fetch';
+
+import { EthereumDefinitions } from '@trezor/transport/lib/types/messages';
+
+// considerations:
```

Comment:
I added this follow up https://github.com/trezor/trezor-suite/issues/7943

With those considerations.